test/CoWSwapEthFlow.t.sol

```diff
@@ -183,3 +216,219 @@ contract TestCoWSwapEthFlow is Test, ICoWSwapOnchainOrders {
         assertEq(ethFlowValidTo, validTo);
     }
 }
+
+contract OrderDeletion is EthFlowTestSetup {
+    using EthFlowOrder for EthFlowOrder.Data;
+    using GPv2Order for GPv2Order.Data;
+    using GPv2Order for bytes;
+
+    struct OrderDetails {
+        EthFlowOrder.Data data;
+        bytes32 hash;
+        bytes orderUid;
+    }
+
+    function orderDetails(EthFlowOrder.Data memory order)
+        internal
+        view
+        returns (OrderDetails memory)
+    {
+        bytes32 orderHash = order.toCoWSwapOrder(wrappedNativeToken).hash(
+            ethFlow.cowSwapDomainSeparatorPublic()
+        );
+        bytes memory orderUid = new bytes(GPv2Order.UID_LENGTH);
+        orderUid.packOrderUidParams(
+            orderHash,
+            address(ethFlow),
+            type(uint32).max
+        );
+        return OrderDetails(order, orderHash, orderUid);
+    }
+
+    function dummyOrder() internal view returns (EthFlowOrder.Data memory) {
+        EthFlowOrder.Data memory order = EthFlowOrder.Data(
+            IERC20(FillWithSameByte.toAddress(0x01)),
+            FillWithSameByte.toAddress(0x02),
+            FillWithSameByte.toUint256(0x03),
+            FillWithSameByte.toUint256(0x04),
+            FillWithSameByte.toBytes32(0x05),
+            FillWithSameByte.toUint256(0x06),
+            FillWithSameByte.toUint32(0x07),
+            true,
+            FillWithSameByte.toUint64(0x08)
+        );
+        require(
+            order.validTo > block.timestamp,
+            "Dummy order is already expired, please update dummy expiration value"
+        );
+        return order;
+    }
+
+    function createOrderWithOwner(OrderDetails memory order, address owner)
+        public
+    {
+        vm.deal(owner, order.data.sellAmount);
+        vm.startPrank(owner);
```

Comment:
I see here start prank and later just prank. It's my impression that these are synonymous. Maybe it's better to just use one everywhere to avoid confusion.

Reply:
You are right, I used them inconsistently and assuming that they are synonymous.
They actually have a different meaning, `prank` applies only to the next call, while `startPrank` applies to all follow-up calls. This was the source of a bug, as I used `prank` with the assumption it would apply to subsequent calls but it applied only to the next, and the error message for a bad caller was the same as the one I was testing for.
Anyway, now that's fixed and the two are used correctly now!